### DISCUSSION TITLE
v5.0.0-rc2

### DISCUSSION THREAD

**Discussion by wisskid** [smarty-php/smarty#916] [General]
## What's Changed
* Fix strip_tags modifier for falsy input. by @wisskid in https://github.com/smarty-php/smarty/pull/893
* Smarty5 by @wisskid in https://github.com/smarty-php/smarty/pull/852
* Fix use of negative numbers in math equations. by @wisskid in https://github.com/smarty-php/smarty/pull/903
* Fix case-sensitive tag names (#907) by @Jack-Dane in https://github.com/smarty-php/smarty/pull/910
* Typos in documentation by @shadowwa in https://github.com/smarty-php/smarty/pull/914
* Do not auto-html-escape custom function results. by @wisskid in https://github.com/smarty-php/smarty/pull/908

## New Contributors
* @Jack-Dane made their first contribution in https://github.com/smarty-php/smarty/pull/910
* @shadowwa made their first contribution in https://github.com/smarty-php/smarty/pull/914

**Full Changelog**: https://github.com/smarty-php/smarty/compare/v4.3.2...v5.0.0-rc2

<hr /><em>This discussion was created from the release <a href='https://github.com/smarty-php/smarty/releases/tag/v5.0.0-rc2'>v5.0.0-rc2</a>.</em>

**Comment by wisskid** [comment]
I've released upgrade instructions for v5 on https://smarty-php.github.io/smarty/5.x/upgrading/

**Reply by saulery** [reply]
Hello,

Quick proofreading. The instructions are clear.

Tow spaces are missing between PHP and version numbers.

I also find occurrences of registered_filters inside src/Debug.php file.

![2023-11-29 14_13_46-Greenshot](https://github.com/smarty-php/smarty/assets/10128284/00f70ecd-f0ac-4176-b974-540adf28e68d)

Regards,

**Reply by wisskid** [reply]
Not in master branch: https://github.com/search?q=repo%3Asmarty-php%2Fsmarty%20registered_filters&type=code

**Comment by saulery** [comment]
We don't talk about smarty5 banch?

https://github.com/smarty-php/smarty/blob/0750fd7f1e376ad61f40ec4cb61cf878aef3e957/src/Debug.php#L213

**Reply by wisskid** [reply]
Ah, no, that branch has already been merged into master. I'll remove it.

**Comment by saulery** [comment]
Ok, then I will update WIKINDX with v5.0.0-rc2 from composer and check again.

**Reply by wisskid** [reply]
Cool, please let us know how it works out!

**Comment by saulery** [comment]
No error with this RC.

Just I noted that inside function getTagCompiler() $tag is lowered twice.

**Reply by wisskid** [reply]
Thanx, removed the 2nd one in master.

**Comment by marclaporte** [comment]
@lkppo 
Time for the real thing!
https://github.com/smarty-php/smarty/discussions/956

**Reply by saulery** [reply]
Yes, I follow all the discussions. I'm testing this evening.

**Reply by saulery** [reply]
WIKINDX updated. All is fine.
